content/posts/April-2020.md

```diff
@@ -59,3 +59,19 @@ draft: false
 [#797]: https://github.com/packit-service/packit/pull/797
 [#788]: https://github.com/packit-service/packit/pull/788
 [#564]: https://github.com/packit-service/packit-service/pull/564
+
+## Week 17 (April 20th - April 24th)
+
+### packit-service
+
```

Comment:
Could you please add a sentence that in this week we mostly focused on centos stream and some under the hood improvements and that none of it is available for github projects?